project/types_yaml_test.go

```diff
@@ -192,3 +197,24 @@ func TestMaporsliceYaml(t *testing.T) {
 	assert.True(t, contains(s2.Foo.parts, "bar=baz"))
 	assert.True(t, contains(s2.Foo.parts, "far=faz"))
 }
+
+var sampleStructCommand = `command: bash`
+
+func TestUnmarshalCommand(t *testing.T) {
+	s := &StructCommand{}
+	err := yaml.Unmarshal([]byte(sampleStructCommand), s)
+
+	assert.Nil(t, err)
+	assert.Equal(t, []string{"bash"}, s.Command.Slice())
+	assert.Nil(t, s.Entrypoint.Slice())
+
+	bytes, err := yaml.Marshal(s)
+	assert.Nil(t, err)
+
+	s2 := &StructCommand{}
+	err = yaml.Unmarshal(bytes, s2)
+
+	assert.Nil(t, err)
+	assert.Equal(t, []string{"bash"}, s.Command.Slice())
+	assert.Nil(t, s.Entrypoint.Slice())
+}
```

Comment:
I don't think this test actually tests the change in behaviour because the command is not empty.